tests/test_benchmarks/test___init__.py

```diff
@@ -423,7 +423,7 @@ class TestVisualDegrees:
         pytest.param('tolias.Cadena2017-pls', 6, '0fe27ddd5b9ea701e380063dc09b91234eba3551', approx(.29641, abs=.0001),
                      marks=[pytest.mark.private_access]),
     ])
-    def test_amount_gray(self, benchmark, candidate_degrees, image_id, expected):
+    def test_amount_gray(self, benchmark, candidate_degrees, image_id, expected, brainio_home, resultcaching_home, brainscore_home):
```

Comment:
I'm not the biggest fan of these configuration parameters in the method, is there a way to not have them here?

Reply:
They're not really parameters, they're fixtures.  Without them the test leaves files behind.  It's a problem when testing locally.  I could [convert](https://docs.pytest.org/en/latest/how-to/fixtures.html#autouse-fixtures-fixtures-you-don-t-have-to-request) those fixtures to `autouse=True` I suppose.